### DISCUSSION TITLE
Is there a way to redirect to preselect some elements of a table?

### DISCUSSION THREAD

**Discussion by NicolasSacC** [filamentphp/filament#4172] [Help] [answered]
Hi!

Is there a way to pre-select some rows of a table (I am creating a "Return" button on a custom page and I don't want the users to lose their selection.

I tried to look through the documentation but found nothing..

**Comment by hrvoj3e** [comment, marked as answer]
Yes. Use sessions to store filters.
Read here
 - https://github.com/filamentphp/filament/discussions/4079#discussioncomment-3679158
